data_portal_api.yml

```diff
@@ -226,11 +377,6 @@ paths:
         This lists all projects and their UUIDs that currently exist in the data portal. If a parameter is specified as
         a filter, then only projects that meet the status criteria will be outputted.
       parameters:
-        - name: status
-          in: query
-          schema:
-            type: string
-            enum: [foo,bar]
         - $ref: '#/components/parameters/user_uuid'
         - name: fromDate
```

Comment:
I think it's more conventional to use underscores? `from_date`? Ditto for toDate.

Reply:
I actually thing for JSON the convention is camelcase over underscores.